Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM registry.access.redhat.com/ubi8/ubi:8.9 AS build-stage0
 
-ARG OC_VERSION="stable"
+ARG OC_VERSION="stable-4.15"
```

Comment:
Are we sure binding it to a particular version?

Reply:
seems . latest version of `oc` required `GLIBC_2.33` and ubi8/ubi:8.9 image may not include the latest `GLIBC` version .. 
It's not nice . we need to create a card to fix the old image :(